dali/base/dadiags.cpp

```diff
@@ -122,7 +123,7 @@ class CDaliDiagnosticsServer: public IDaliServer, public Thread
                     mb.append(getReceiveQueueDetails(buf).str());
                 }
                 else if (0 == stricmp(id, "locks")) {
-                    mb.append(querySDS().getLocks(buf).str());
+                    querySDS().getLocks(mb);
```

Comment:
This is server side which should return a serialized format (in MB). The other end is expecting the serialized format (in MB) and will convert it to plain text. Not sure I understand your comment correctly.